octopus.Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM opensuse/leap:15.1
```

Comment:
Wonder if we should use 15.2, sesdev uses 15.2 Vagrant box for octopus even it's not GA yet.

Reply:
I'm just not sure which python version 15.2 will  have when its release, so its more safe to use 15.1.

Reply:
I'm currently building Ceph, so 15.1 seems to be ok and no
```
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
Could NOT find Python3 (missing: Development) (found suitable version
"3.8.2", minimum required is "3")
```
happened.

Reply:
I agree with @bk201, we should use leap15.2. Let's be consistent with `sesdev` - https://github.com/SUSE/sesdev/blob/master/seslib/__init__.py#L116

Reply:
I have updated to 15.2.
My previous concern doesn't matter anymore, since the error only happens when you compile on top of an existing 3.6 build.